polygon/erc20/tokens.sql

```diff
@@ -199,6 +199,1058 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
 \\x08e6d1f0c4877ef2993ad733fc6f1d022d0e9dbf	jPLN	18
 \\xa22f6bc96f13bcc84df36109c973d3c0505a067e	jKRW	18
 \\x6a304dFdb9f808741244b6bfEe65ca7B3b3A6076	PTaUSDC	6
+\\xd6a5ab46ead26f49b03bbb1f9eb1ad5c1767974a EMON 18
```

Comment:
@yongjun925 thank you for the PR. each row you added appear to have spacing issue. assuming it's a single space between each field, that won't work. copy a prior row to get the right format (i.e. line 201). that should be applied to all new rows. hope that makes sense. once changed, i can add quickly